### PR TITLE
feat: add default appProtocol on oc receiver

### PR DESCRIPTION
**Pull request by husnialhamdani**
**Description:** <Describe what has changed.>
add missing appProtocol in opencensus receiver

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-operator/issues/2487

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: .chloggen/main.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: add missing appProtocol in opencensus receiver
+
+# One or more tracking issues related to the change
+issues: []
```
Comment (swiatekm-sumo) [inline .chloggen/main.yaml:11]: You need to put your issue number here.

Comment (swiatekm-sumo) [inline .chloggen/main.yaml:11]: You need to put your issue number here.

**Review by swiatekm-sumo — COMMENTED** [review]
Please make the linters happy. Otherwise LGTM :+1:

File: .chloggen/main.yaml (hunk 1 above)
Comment (husnialhamdani) [reply .chloggen/main.yaml]: Added, thanks.

File: internal/manifests/collector/parser/receiver/receiver_oc.go [hunk 2]
```diff
@@ -24,12 +24,14 @@ const parserNameOpenCensus = "__opencensus"
 
 // NewOpenCensusReceiverParser builds a new parser for OpenCensus receivers.
 func NewOpenCensusReceiverParser(logger logr.Logger, name string, config map[interface{}]interface{}) parser.ComponentPortParser {
+	http := "http"
```
[the remark this answers is not in the document]
Comment (swiatekm-sumo) [reply internal/manifests/collector/parser/receiver/receiver_oc.go:27]: You need to change the variable name, to `httpAppProtocol` for example.

**Review by swiatekm-sumo — APPROVED** [review]

**Review by jaronoff97 — APPROVED** [review]

**Pull request merged**